governance/parameter-docs/param-dai-savings-rate.md

```diff
@@ -0,0 +1,50 @@
+# Dai Savings Rate
+
+```
+Alias: DSR
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs: https://docs.makerdao.com/mcd-developer-guides/developer-guides-and-tutorials#dai-savings-rate-dsr
+```
+
+## Description
+The Dai Savings Rate is a tool that Maker Governance can use to adjust the interest rate paid to DAI holders who have deposited DAI into the Dai Savings Rate contract (`Pot`). Maker Governance directly controls the Dai Savings Rate.
+
+The Dai Savings Rate is usually expressed as a percentage and can be set to any value. However, even though Maker Governance can set the Dai Savings Rate to a negative value, this will not affect the balances held within the `Pot`. The balances within the `Pot` will not decrease because the drip function will revert if negative.
+
+## Purpose
+
+The Dai Savings Rate aims to enable Maker Governance to encourage DAI holders to deposit DAI within the `Pot`. Higher amounts of DAI within the `Pot` should reduce the market supply of DAI and act as upwards pressure on the DAI peg.
+
+## Trade-offs
+
+Increasing the Dai Savings Rate is a trade-off between increased DAI demand and decreased DAI liquidity and may also result in increased expenses for the Maker Protocol.
```

Comment:
There's no 'may' here really. The only case in which this does not increase expenses for the protocol is if there is exactly zero DAI locked in the DSR.

Reply:
Also, try to be super clear about which way the trade-off goes. Increasing the Dai Savings Rate increases DAI demand at the cost of increasing protocol expenses, and decreasing DAI liquidity.